Logix/Users/GetIsUserScaling.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FrooxEngine.LogiX.Users
+{
+    public class GetScalingEnabled
+    {
+        [NodeName("Get User Scaling")]
```

Comment:
The filename says `GetIsUserScaling`, the class name says `GetScalingEnabled` and the node name says `Get User Scaling`. I think all of those should be the same, and class name makes most sense as the others sound more like you are getting if the user is actively changing their scale right now which is not what that node does

Reply:
Done